agents/ls-csharp/src/main/resources/org.eclipse.che.ls.csharp.script.sh

```diff
@@ -13,7 +13,7 @@ unset PACKAGES
 unset SUDO
 command -v tar >/dev/null 2>&1 || { PACKAGES=${PACKAGES}" tar"; }
 command -v curl >/dev/null 2>&1 || { PACKAGES=${PACKAGES}" curl"; }
-test "$(id -u)" = 0 || SUDO="sudo -E"
+test "$(id -u)" = 0 || test -f ${HOME}/is_arbitrary_user || SUDO="sudo -E"
```

Comment:
can we have more details about this test ?
commit message being : "Modifications to the shell scripts that bootstrap the servers"